translator/translate/otel/processor/ec2taggerprocessor/translator.go

```diff
@@ -65,7 +65,8 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	}
 
 	cfg.MiddlewareID = &agenthealth.StatusCodeID
-	cfg.RefreshIntervalSeconds = time.Duration(0)
+	cfg.RefreshTagsInterval = time.Duration(0)
+	cfg.RefreshVolumesInterval = 5 * time.Minute
```

Comment:
Are we sure we want to enable this by default? If volumes aren't configured on the ec2tagger, then why are we refreshing? Shouldn't this only be if the `AttributeVolumeId` is configured in the block above?